src/rebar3_proper_prv.erl

```diff
@@ -237,6 +240,12 @@ proper_opts([{noshrink, true} | T]) -> [noshrink | proper_opts(T)];
 proper_opts([{noshrink, false} | T]) -> proper_opts(T);
 proper_opts([{any_to_integer, true} | T]) -> [any_to_integer | proper_opts(T)];
 proper_opts([{any_to_integer, false} | T]) -> proper_opts(T);
+proper_opts([{on_output, {Mod, Fun}} | T]) ->
+    [{on_output, fun Mod:Fun/2} | proper_opts(T)];
+proper_opts([{on_output, MFStr} | T]) when is_list(MFStr) ->
+    {ok, Tokens, _}  = erl_scan:string(MFStr++"."),
+    {ok, {Mod, Fun}} = erl_parse:parse_term(Tokens),
+    [{on_output, {Mod, Fun}} | proper_opts(T)];
```

Comment:
Correct. PropEr expects a fun.It worked for me, but there's no reason not to pass it back through `proper_opts`. Can you think of a better way to specify the fun here?